README.md

```diff
@@ -255,6 +274,32 @@ Add the required kodyfire metadata to your package.json
 
 Intialize your project as a git repository and push to a public Github repo
 
+To do so, kindlly follow these steps:-
```

Comment:
Review the spelling: 
Initialize your project as git repository and push to your public Github repository by following these steps:

1. Initialize a new Github repository and make it public
2. Open your project root folder locally from the terminal and run the following commands: 
```bash
 git init
git add .
git commit -m 'initial commit'
```
3. Link your project to your Github repository
```bash
git remote add origin <copied URL>
git push origin main
```